crates/ruff_linter/src/rules/flake8_commas/rules/trailing_commas.rs

```diff
@@ -98,6 +100,8 @@ enum ContextType {
     Dict,
     /// Lambda parameter list, e.g. `lambda a, b`.
     LambdaParameters,
+    /// Type parameter list, e.g. `[T, U]`
```

Comment:
nit: we might want a slightly longer example here. This just looks like a list.


```suggestion
    /// Type parameter list, e.g. `def foo[T, U](): ...`
```